test/e2e/redundancy/gateway_failover.go

```diff
@@ -85,6 +85,12 @@ func testGatewayPodRestartScenario(f *subFramework.Framework) {
 		ToEndpointType:        defaultEndpointType(),
 	})
 
+	if framework.TestContext.GlobalnetEnabled {
```

Comment:
If this PR passes Globalnet+K8s 1.20 CI jobs, instead of doing changes in Submariner repo - the other option I was thinking was to delete the Service and ServiceExport at the end of CreatePods API in Shipyard - https://github.com/submariner-io/shipyard/blob/4a7b223ec874877364f35debcd4d45739cfe3f1b/test/e2e/tcp/connectivity.go#L169

Reply:
Hurrah!! The Globalnet jobs are now passing in CI. 
@tpantelis @dfarrell07 @mangelajo what are your thoughts on my suggestion above? Shall we move this fix to Shipyard CreatePods API so that client code (aka Submariner redundancy tests) will be transparent to the changes?

Reply:
That sounds fine but can you explain the reason for the failure and why this fixes it?

Reply:
Sure @tpantelis. I'll push a PR in Shipyard and explain the situation in commit message.

PS: I re-checked the CI tests and it passed once again :)

Reply:
The following PR is pushed in shipyard - https://github.com/submariner-io/shipyard/pull/530

Reply:
I'll close this PR.

Reply:
@tpantelis @mangelajo @skitt Instead of closing this PR, I modified it to consume the latest Shipyard repo. Can you please take look. Thanks.